<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kcal</class>
 <widget class="QWidget" name="Kcal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 96, 0, 255), stop:1 rgba(255, 255, 255, 255));
</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>610</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>460</width>
     <height>610</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>460</width>
     <height>610</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(104, 161, 53, 255), stop:1 rgba(255, 255, 255, 255));

</string>
   </property>
   <widget class="QWidget" name="page">
    <property name="minimumSize">
     <size>
      <width>460</width>
      <height>610</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>460</width>
      <height>610</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(104, 161, 53, 255), stop:1 rgba(255, 255, 255, 255));

</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
font: 30pt &quot;MS Shell Dlg2&quot;;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Calculator KCAL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_back_to_menu">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>530</y>
       <width>280</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>280</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:grey;
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
     </property>
     <property name="text">
      <string>Back to menu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;
font:22px;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Your gender: </string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_gender">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">FONT-SIZE:24PX;
background-color: WHITE;</string>
     </property>
     <item>
      <property name="text">
       <string>MEN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WOMEN</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_calculate_kcal">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>430</y>
       <width>240</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 170, 0);
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;
font:22px;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Your height [cm]: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;
font:22px;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Your weight [kg]: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;
font:22px;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Your age [year]: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>210</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">FONT-SIZE:24PX;
background-color: WHITE;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>270</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">FONT-SIZE:24PX;
background-color: WHITE;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>330</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">FONT-SIZE:24PX;
background-color: WHITE;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_activity">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">FONT-SIZE:17PX;
background-color: WHITE;</string>
     </property>
     <item>
      <property name="text">
       <string>1. Lack of physical activity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2. Low level physical activity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3. Training twice a week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4. Training 3-4 times a week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5. Training 5 times a week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6. Very hard physical work</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;
font:22px;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Your activity: </string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
